Sources/Wrap.swift

```diff
@@ -361,11 +367,13 @@ extension Date: WrappableDate {
 }
 
 /// Extension making NSdate a WrappableDate
-extension NSDate: WrappableDate {
-    public func wrap(dateFormatter: DateFormatter) -> String {
-        return dateFormatter.string(from: self as Date)
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
Same feedback as above, use a simpler conditional, and unstage the indentation changes.